services/server/test/helpers/assertions.ts

```diff
@@ -4,15 +4,29 @@ import config from "config";
 import path from "path";
 import fs from "fs";
 import { getAddress, id } from "ethers";
-import { getMatchStatus } from "../../src/server/common";
 import type { Response } from "superagent";
 import type { Done } from "mocha";
 import { Pool } from "pg";
 import {
   Transformation,
   TransformationValues,
+  VerificationStatus,
 } from "@ethereum-sourcify/lib-sourcify";
 import { ServerFixture } from "./ServerFixture";
+import { Match } from "../../src/server/types";
+
+function getMatchStatus(match: Match): VerificationStatus {
```

Comment:
I think we shouldn't copy this function to here but use the one suggested in my other comment.